libqtile/backend/x11/xcore.py

```diff
@@ -397,14 +397,21 @@ def grab_key(self, key: config.Key) -> Tuple[int, int]:
         keysym, modmask = self.lookup_key(key)
         code = self.conn.keysym_to_keycode(keysym)
 
-        for amask in self._auto_modmasks():
-            self.conn.conn.core.GrabKey(
-                True,
-                self._root.wid,
-                modmask | amask,
-                code,
-                xcffib.xproto.GrabMode.Async,
-                xcffib.xproto.GrabMode.Async,
+        if code != 0:
```

Comment:
Can we just return above once we find out `code`, vs. emitting the error 4 times?

Reply:
I am not sure I understand you; when I tested this the error was only emitted once. Am I overlooking something?

Reply:
Ah, I misread it, sorry for the noise :)

Reply:
Ok, then I'll click `resolve` on this.